test.ts

```diff
@@ -786,3 +786,91 @@ Deno.test("outputToObjectArrayEmpty", function () {
     "Result is not an array or does not have the correct length",
   );
 });
+
+Deno.test("veryLargeAndVerySmall", function () {
+  const db = new DB();
+
+  // return
+
+  const [[selectedPositiveInfinity]] = db.query("SELECT +8e8888");
+  assertEquals(
+    selectedPositiveInfinity,
+    +Infinity,
+    "Failed to return correct value for +Infinity.",
+  );
+
+  const [[selectedNegativeInfinity]] = db.query("SELECT -8e8888");
+  assertEquals(
+    selectedNegativeInfinity,
+    -Infinity,
+    "Failed to return correct value for -Infinity.",
+  );
+
+  const [[selectedPositiveZero]] = db.query("SELECT +1/8e8888");
+  assert(
+    Object.is(selectedPositiveZero, +0),
+    "Failed to return correct value for +0.",
+  );
+
+  const [[selectedNegativeZero]] = db.query("SELECT -1/8e8888");
+  assert(
+    Object.is(selectedNegativeZero, -0),
+    "Failed to return correct value for -0.",
+  );
+
+  const [[selectedPositiveMassive]] = db.query("SELECT +20e20");
+  assertEquals(
+    selectedPositiveMassive,
+    +20e20,
+    "Failed to return correct value for +20e20.",
+  );
+
+  const [[selectedNegativeMassive]] = db.query("SELECT -20e20");
+  assertEquals(
+    selectedNegativeMassive,
+    -20e20,
+    "Failed to return correct value for -20e20.",
+  );
+
+  // bind
```

Comment:
The comment is not necessary after removing the preceding section 😅

Reply:
Removed, done.